### PR TITLE
Adding figure gh pages builds to contrib docs

### PR DESCRIPTION
**Pull request by hflynn**
Now there are CI builds for the figure website, they need documenting so everyone knows how to update the pages.

--no-rebase

### CODE REVIEW

**Comment by hflynn** [comment]
Will be staged on https://www.openmicroscopy.org/site/support/contributing-staging/ci-docs.html and https://www.openmicroscopy.org/site/support/contributing-staging/jekyll.html

File: contributing/ci-docs.txt [hunk 1]
```diff
@@ -342,6 +357,37 @@ The repository for the OME help is https://github.com/openmicroscopy/ome-help.
...
+		.. seealso:: :doc:`jekyll`
+
+	:jenkinsjob:`FIGURE-help-release`
+
+		This job is used to deploy the OME help documentation
+
+		#. Opens a Pull Request from
+		   https://github.com/openmicroscopy/figure/tree/gh-pages-staging
+		   to https://github.com/openmicroscopy/figure/tree/gh-pages. If
+		   this PR is merged, the GitHub Pages services updates the content of
```
Comment (mtbc) [inline contributing/ci-docs.txt]: "he GitHub Pages services updates" -- should it be "services update" or "service updates"?

File: contributing/ci-docs.txt [hunk 2]
```diff
@@ -342,6 +357,37 @@ The repository for the OME help is https://github.com/openmicroscopy/ome-help.
...
+		   http://figure.staging.openmicroscopy.org
+
+		.. seealso:: :doc:`jekyll`
+
+	:jenkinsjob:`FIGURE-help-release`
+
+		This job is used to deploy the OME help documentation
+
+		#. Opens a Pull Request from
+		   https://github.com/openmicroscopy/figure/tree/gh-pages-staging
```
Comment (sbesson) [inline contributing/ci-docs.txt]: `https://github.com/ome/figure/tree/gh-pages-staging`

File: contributing/ci-docs.txt [hunk 3]
```diff
@@ -342,6 +357,37 @@ The repository for the OME help is https://github.com/openmicroscopy/ome-help.
...
+		   to https://github.com/openmicroscopy/figure/tree/gh-pages. If
```
Comment (sbesson) [inline contributing/ci-docs.txt]: `https://github.com/ome/figure/tree/gh-pages`

File: contributing/ci-docs.txt [hunk 4]
```diff
@@ -342,6 +357,37 @@ The repository for the OME help is https://github.com/openmicroscopy/ome-help.
 			#. rysnc the content of :file:`/ome/data_repo/public/help-staging`
 			   to :file:`/ome/data_repo/public/help`
 
+OMERO.figure website
+^^^^^^^^^^^^^^^^^^^^
+
+The repository for OMERO.figure is https://github.com/openmicroscopy/figure.
```
Comment (sbesson) [inline contributing/ci-docs.txt]: `https://github.com/ome/figure`

**Comment by hflynn** [comment]
@mtbc @sbesson should be fixed now - staging pages rebuilt ready for checking

**Comment by sbesson** [comment]
https://ci.openmicroscopy.org/view/Docs/job/CONTRIBUTING-merge-docs/537/ green. :+1: Anything else?

**Comment by hflynn** [comment]
I missed two @mtbc should be fine across the whole page now

**Comment by mtbc** [comment]
Those too look like good fixes. :smiley:

**Pull request merged**